File: C++/A/999A-mishkaandcontest.cpp
#include <bits/stdc++.h>
using namespace std;

class Solver
{
    int n, k;
    deque<int> problems;

public:
    void input()
    {
        cin >> n >> k;
        for (int i = 0; i < n; i++)
        {
            int x;
            cin >> x;
            problems.push_back(x);
        }
    }
    int answer()
    {
        int solvable = 0;
        for (int i = 0; i < n; i++)
        {
            if (problems[i] > k)
                break;
            ++solvable;
        }
        for (int i = 0; i < n; i++)
        {
            if (problems[n - 1 - i] > k)
                break;
            ++solvable;
        }
        return min(n, solvable);
    }
};

int main()
{
    Solver solution;
    solution.input();
    cout << solution.answer();
}

File: Python/A/999A-mishkaandcontest.py
class Solver:
    def __init__(self):
        self.n, self.k = 0, 0
        self.problems = []

    def input(self):
        self.n, self.k = map(int, input().split())
        self.problems = list(map(int, input().split()))

    def answer(self):
        solvable = 0
        for i in range(self.n):
            if self.problems[i] > self.k:
                break
            solvable += 1
        for i in range(self.n):
            if self.problems[self.n - 1 - i] > self.k:
                break
            solvable += 1
        return min(self.n, solvable)


solution = Solver()
solution.input()
print(solution.answer())
